### PR TITLE
GW-39: Develop about section to be responsive

### PR DESCRIPTION
**Pull request by chrishadrian**
- Added responsiveness for about section
Preview: https://deploy-preview-15--gisaubcv3.netlify.app/

### CODE REVIEW

File: src/pages/home/components/About.js [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+      <a href="/about" className="sm:w-1/3 2xl:w-fit sm:absolute sm:right-1/10">
+        <img
+          src={isMobile ? about_mobile : about_image}
+          alt="GISAU Executives"
+          className="w-full"
+        />
+      </a>
+      <Box className="w-[60%] sm:w-1/2 h-full">
         <p className={typographyStyle}>
           Hello! We are a non-profit cultural organization centered in promoting
```
Comment (Julian-UBC) [inline src/pages/home/components/About.js:36]:
```suggestion
          Halo! We are a non-profit cultural organization centered in promoting
```

File: src/shared/components/HomeButton.js [hunk 2]
```diff
@@ -5,16 +5,25 @@ const useStyles = makeStyles({
...
-        paddingTop: "0.3rem",
+        height: "2.5rem",
+        padding: "0",
         transition: "background 300ms ease-out",
         "&:hover": {
             background: "linear-gradient(to left, #7D0202, #B25F4C)",
         },
-        marginTop: "0.5rem"
+        marginTop: "0.5rem",
+        "@media (min-width: 1500px)": {
```
Comment (Julian-UBC) [inline src/shared/components/HomeButton.js:18]: Any reasoning or sources for using this screen width?

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
I will review further later! Here are some obvious changes for now.

File: src/shared/components/HomeButton.js (hunk 2 above)
Comment (chrishadrian) [reply src/shared/components/HomeButton.js:18]: tailwind's 2xl is 1536. will change to that.

File: src/pages/home/components/About.js (hunk 1 above)
Comment (chrishadrian) [reply src/pages/home/components/About.js:36]: Are we sure we want to use Halo? 
image

File: src/shared/components/HomeButton.js (hunk 2 above)
Comment (chrishadrian) [reply src/shared/components/HomeButton.js:18]: Going to change this to Tailwind's '2xl', which is 1536p

**Review by jenaarianto — APPROVED** [review]

File: src/shared/components/HomeButton.js (hunk 2 above)
Comment (Julian-UBC) [reply src/shared/components/HomeButton.js:18]: Okayy

File: src/pages/home/components/About.js (hunk 1 above)
Comment (Julian-UBC) [reply src/pages/home/components/About.js:36]: @jenaarianto any thoughts?? I feel like we need to include an introduction in bahasa indo so Halo should b fine.

Comment (chrishadrian) [reply src/pages/home/components/About.js:36]: Changed to Halo after a thorough discussion

**Review by Julian-UBC — APPROVED** [review]

**Pull request merged**